src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
There seems to be a syntax error due to a missing closing quote for the string. Make sure to close the string with a matching quote to fix the syntax error.